sermons/index.html

```diff
@@ -56,6 +56,11 @@ <h2>All sermons</h2>
             I will pass!
             <footer>- <a href="https://www.twitch.tv/daut/clip/ScrumptiousCourageousMuleDAESuppy">DauT</a></footer>
          </blockquote>
+         
+         <blockquote>
+            Am I a Wizard? Not... I am freaking Gandalf!!!
+            <footer>- <a href="https://www.twitch.tv/daut/clip/ScrumptiousCourageousMuleDAESuppy">DauT</a></footer>
```

Comment:
This is the "I will pass!" clip, not a clip with "Am I a Wizard? Not... I am freaking Gandalf!!!" in it